projects/step-frontend/src/lib/modules/timeseries/performance-view/filter-bar/filter-bar.component.scss

```diff
@@ -47,7 +87,59 @@
 }
 
 .apply-btn {
-  background-color: var.$yellow-10;
-  border: 1px solid var.$yellow-600;
-  color: var.$yellow-650;
+  //background-color: var.$warn-yellow-background;
+  //border: 1px solid var.$warn-yellow-border;
+  //color: var.$warn-yellow-color;
```

Comment:
I would remove the commeted out section, if we need them again they are documented in this PR